synapseclient/client.py

```diff
@@ -3407,7 +3407,9 @@ def restGET(self, uri, endpoint=None, headers=None, retryPolicy={}, **kwargs):
         Performs a REST GET operation to the Synapse server.
 
         :param uri:      URI on which get is performed
-        :param endpoint: Server endpoint, defaults to self.repoEndpoint
+        :param endpoint: Server endpoint, defaults to 'https://repo-prod.prod.sagebase.org/file/v1'.  Some
```

Comment:
The endpoint does not always default to 'https://repo-prod.prod.sagebase.org/file/v1'
users can change the endpoint using `syn.setEndpoints(**synapseclient.client.STAGING_ENDPOINTS)`, in which case `endpoint` would default to the staging endpoint instead of production endpoint

Reply:
@zimingd  that is a good point.  How would you write the Py client documentation so that someone could use 'restGET' along with the APIs doc's?

Reply:
Instead of listing the URLS you can mention the two constant dicts synapseclient.client.STAGING_ENDPOINTS
synapseclient.client.PRODUCTION_ENDPOINTS

and have then access the defiend endpoints like 

```
syn.restGET('/some/api/', endpoint=synapseclient.client.PRODUCTION_ENDPOINTS['fileHandleEndpoint'])
```

Or if always using the full name is too long
```
from synapseclient.client import PRODUCTION_ENDPOINTS
syn.restGET('/some/api/', endpoint=PRODUCTION_ENDPOINTS['fileHandleEndpoint'])
```

The available keys in PRODUCTION_ENDPOINTS and STAGING_ENDPOINTS are:
```
fileHandleEndpoint
repoEndpoint
authEndpoint
portalEndpoint
```